examples/configuration/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { DaprClient, /* CommunicationProtocolEnum */ } from "dapr-client";
```

Comment:
It would be good to keep the examples consistent. Something like this. @XavierGeerinck Any thoughts on this?
https://docs.dapr.io/developing-applications/building-blocks/configuration/howto-manage-configuration/

Reply:
I think this will create more confusion so I would not write it like that